recipe/meta.yaml

```
@@ -1,34 +1,53 @@
-{% set version = "1.4.2" %}
+{% set name = "joblib" %}
+{% set version = "1.5.1" %}
+{% set tests_to_skip = "test__get_items" %}
+{% set tests_to_skip = tests_to_skip + " or test__get_items_to_delete" %}
+{% set tests_to_skip = tests_to_skip + " or test_call_and_shelve_lazily_load_stored_result" %}
+{% set tests_to_skip = tests_to_skip + " or test_memory_reduce_size_bytes_limit" %}
+{% set tests_to_skip = tests_to_skip + " or test_memory_reduce_size_items_limit" %}
+{% set tests_to_skip = tests_to_skip + " or test_memory_reduce_size_age_limit" %}
+{% set tests_to_skip = tests_to_skip + " or test_parallel_with_unpicklable_functions_in_args" %} # [win64]
```

```suggestion
{% set tests_to_skip = tests_to_skip + " or test_parallel_with_unpicklable_functions_in_args" %} # [win]
```
Keep `win` for simplicity because we already do not support win-32